01_fib_series/fibseries.go

```diff
@@ -0,0 +1,20 @@
+package main
```

Comment:
Tests missing.

Reply:
So added testing and have updated all review comments.